modules/migration-setting-up-target-cluster-to-accept-source-dns-domain.adoc

```diff
@@ -1,34 +1,35 @@
 // Module included in the following assemblies:
 //
 // * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-mtc.adoc
 
 [id="setting-up-target-cluster-to-accept-source-dns-domain_{context}"]
-== Setting up the target cluster to accept the source DNS domain
+= Setting up the target cluster to accept the source DNS domain
 
-You can set up the target cluster to accept requests for migrated applications in the DNS domain of the source cluster.
+You can set up the target cluster to accept requests for a migrated application in the DNS domain of the source cluster.
 
 .Procedure
 
 For both non-secure HTTP access and secure HTTPS access, perform the following steps:
 
-. Create a route in the application’s project for the hostname that applications used in the source cluster:
+. Create a Route in the target cluster's project that is configured to accept requests addressed to the application's FQDN in the source cluster:
```

Comment:
Why is "Route" capitalized here and nowhere else? I think you can use lower-case because the procedure is not CR-specific.